Adding more examples of labels. (#346)
cfrg/draft-irtf-cfrg-voprf: draft-irtf-cfrg-voprf.md

in a Python-like pseudocode. The data types `PrivateInput` and `PublicInput`
are opaque byte strings of arbitrary length no larger than 2^13 octets.

> String values such as 'DeriveKeyPair', 'Seed-', or "Finalize" are ASCII string literals.

Reviewer: ```suggestion
String values such as "DeriveKeyPair", "Seed-", and "Finalize" are ASCII string literals.
```
